### DISCUSSION TITLE
Overview page of open PRs to show the out-of-date (or rebased) state of a PR

### DISCUSSION THREAD

**Discussion by tinuroth** [community/community#11856] [Pull Requests]
The list of open PRs shows quite a bit of information about the PRs already. Author, title, date, people assigned, reviewers/commenters, build/test status (e.g. by virtue of CI/CD integration), and even the state of approval (details about reviewers is only available in the tooltip), plus all labels set. This has improved and matured much over time, which is great.

What is missing, though, is the information whether or not a PR is rebased, or out-of-date (in need of rebasing before merging, along our process). For this information, I have to open the PR page and scroll all the way down to the bottom. Can this information be made available in the overview page of open PRs? Is this configurable already? If not, can it be made configurable, or even a general feature?

**Comment by DiskoRico** [comment]
If not having it in the overview page.
How can I easily filter out all PRs that their branch is outdated with the base branch?
